File: Data Structures & Algorithms/k-closest-points-to-origin/submission-1.java
class Solution {
    public int[][] kClosest(int[][] points, int k) {
        PriorityQueue<int[]> q = new PriorityQueue<>((a, b) -> {
            int d1 = a[0] * a[0] + a[1] * a[1];
            int d2 = b[0] * b[0] + b[1] * b[1];
            return d2 - d1;
        });

        for (int[] point : points) {
            q.offer(point);

            while (q.size() > k) {
                q.poll();
            }
        }
        int[][] ans = new int[k][2];

        for (int i = 0; i < k; i++) {
            ans[i] = q.poll();
        }

        return ans;
    }
}

File: Data Structures & Algorithms/is-anagram/submission-1.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        a1 = dict()
        for a in s:
            a1[a] = a1.get(a, 0) + 1
        
        a2 = dict()
        for b in t:
            a2[b] = a2.get(b, 0) + 1

        return a1 == a2
        